core/flyout_horizontal.js

```diff
@@ -42,6 +41,12 @@ class HorizontalFlyout extends Flyout {
   constructor(workspaceOptions) {
     super(workspaceOptions);
     this.horizontalLayout = true;
+    this.contentWidth_ = 0;
+    window.addEventListener('resize', () => {
+      if (this.RTL) {
+        this.reflowInternal_();
+      }
+    });
```

Comment:
I think this should use conditionalBind method. So something like: 

```
this.flyoutResizeHandler_ =
      browserEvents.conditionalBind(window, 'resize', null, function() {});
```
And then we need to dispose of it in the `dispose` method: 

```
browserEvents.unbind(this.flyoutResizeHandler_);
```